forcedotcom/dev-scripts
# fix: remove eslint-plugin-prettier

> requires https://github.com/forcedotcom/eslint-config-salesforce-typescript/pull/31

remove eslint-plugin-prettier
generally bump deps
add eslint-config-prettier

rollout of change from
https://github.com/forcedotcom/eslint-config-salesforce/issues/19

how to test:
Make your dependencies, prettier, and eslint look like this repo (includes remove the eslint-plugin-prettier).  There's several major version bumps in here and it's better to be safe than for dev-scripts to start breaking everything.

Also worth verifying, possibly via yarn-link: I've not removed something from dev-scripts before.  I don't know if install always throws, or removes the plugin, or just does nothing.  If it's bad, let's release this as a major version so it doesn't start automerging.

[@W-11590389@](https://gus.lightning.force.com/a07EE000013zWxUYAU)
